src/main/java/frc/robot/Robot.java

```diff
@@ -29,7 +29,7 @@
  * project.
  */
 public class Robot extends LoggedRobot {
-  public static final boolean isCompetition = true;
+  public static final boolean isCompetition = false;
```

Comment:
Put back to true once we are ready to push this PR

Reply:
alright

Reply:
Seems like that time is nigh. :)  Let's change to true